includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -116,7 +118,7 @@ public function setup_wizard() {
 		) );
 
 		if ( ! empty( $_POST['save_step'] ) && isset( $this->steps[ $this->step ]['handler'] ) ) {
-			call_user_func( $this->steps[ $this->step ]['handler'] );
+			call_user_func( $this->steps[ $this->step ]['handler'], $this->get_next_step_link() );
```

Comment:
Ah I see, it's so the callback can be external.

Still not totally happy about passing around links like that - you could pass `$this` to the callback and have public methods if this is needed externally.

@claudiosanches @claudiulodro (let's discuss before making changes)